chart/values.yaml

```diff
@@ -115,8 +129,8 @@ configData:
 
 securityContext:
   enabled: true
-  runAsUser: 1000
-  fsGroup: 1000
+  runAsUser: 0
```

Comment:
is it required to run everything under root?

Reply:
It is needed to get access to /etc/.

Reply:
Fixed